cloud/deploys/dev_azure/README.md

```diff
@@ -3,38 +3,62 @@ This is a deploy directory that allows individuals to create a deploy of the
 app. We intentionally don't use a shared state with this terraform directory
 to allow for everyone to have their own app. 
 
-The process to deploy is similar
-to the staging process but we do not have to do the shared state set up. It 
-should be `terraform init` and then `terraform apply`
+# Running the dev deploy
+## Copy the civiform_config.example.sh 
+Copy the civiform_config.example.sh into civiform_config.sh and
+ change the required variables. 
 
-# Local Docker Build to Remote Azure Deploy
-If you want to do local onto terraform we build/tag/deploy the docker image 
-and then update the azure app service to point to the local image. 
+## Get access to correct technologies
+You will need to reach out to team members to get accounts for the following:
+- Azure
+- AWS
+- Login Radius Civiform-Staging 
 
-## 1. Build, Tag and Push the Docker Image
-This should take like 30 minutes (the push takes the longest).
+## Setup Login Radius For Local Development
+Go to [Login Radius Dashboard](https://dashboard.loginradius.com/) and click
+configure a civiform integration. Choose the outbound SSO Saml.
+
+From there add an app with Sp initiated login and pick a name (this gets put
+into the config as `LOGIN_RADIUS_SAML_APP_NAME`).
 
+To generate the private key for the form run, cat the file and put into dashboard.
 ```
-docker build -f prod.Dockerfile -t <IMAGE_TAG> --cache-from docker.io/civiform/civiform-browser-test:latest --build-arg BUILDKIT_INLINE_CACHE=1 .
-docker tag <IMAGE_TAG> <DOCKER_USERNAME>/<DOCKER_REPO_NAME>:<IMAGE_TAG>
-docker push <DOCKER_USERNAME>/<DOCKER_REPO_NAME>:<IMAGE_TAG>
+openssl genrsa -out private.key 2048
 ```
 
-## 2. Update the image name/tag for your remote azure deploy
-You can do this one of two ways. Terraform deploy or update via the azure 
-portal 
+For generating the cert run, cat the file and put into dashboard.
+```
+openssl req -new -x509 -key private.key -out certificate.cert -days 365 -subj /CN=civiform-staging.hub.loginradius.com
+```
 
-### Update via the .tfvars File and deploy
-After that you can change your .tfvars file to point to the docker tag you set
+We need to copy the details from a previous working setup in login radius 
+once we set up the certs so look back at the staging one to fill out.
```

Comment:
There's also a "service provider" cert generated by the setup script.  I assume that has to be added as well?

Also, we should document the login radius attributes somewhere, so new deployments know what is needed

Reply:
yeah that should probs go in the documentation/run books